pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1982,6 +1982,12 @@ type GaneshaServerSpec struct {
 	// LogLevel set logging level
 	// +optional
 	LogLevel string `json:"logLevel,omitempty"`
+
+	// Whether host networking is enabled for the Ganesha server. If not set, the network settings from the cluster CR will be applied.
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// +nullable
+	// +optional
+	HostNetwork *bool `json:"hostNetwork,omitempty"`
```

Comment:
For readability of a boolean, this is preferred:
```suggestion
	HostNetworkEnabled *bool `json:"hostNetworkEnabled,omitempty"`
```

Reply:
I was preferring "hostNetwork" for consistency in other parts of the project, most specifically #10491 -- your call, though, I'm happy to make the change if that's what you'd prefer. Wouldn't changing that require changing the name in the CRD as well? I'm not super clear on the relationships there

Reply:
True, let's keep it as you have it for consistency with the object store spec. This pattern should have been applied there in the first place, but oh well.